packages/albatross/albatross.1.0.0/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "2.0"
+maintainer: "Hannes Mehnert <hannes@mehnert.org>"
+authors: ["Hannes Mehnert <hannes@mehnert.org>"]
+homepage: "https://github.com/roburio/albatross"
+dev-repo: "git+https://github.com/roburio/albatross.git"
+bug-reports: "https://github.com/roburio/albatross/issues"
+license: "ISC"
+
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune"
+  "dune-configurator"
+  "conf-pkg-config" {build}
+  "lwt" {>= "3.0.0"}
+  "ipaddr" {>= "4.0.0"}
+  "cstruct"
+  "logs"
+  "rresult"
+  "bos"
+  "ptime"
+  "cmdliner" {>= "1.0.0"}
+  "fmt"
+  "astring"
+  "jsonm"
+  "x509" {>= "0.11.0"}
+  "tls" {>= "0.12.2"}
+  "mirage-crypto-pk"
+  "mirage-crypto-rng" {>= "0.8.0"}
+  "asn1-combinators" {>= "0.2.0"}
+  "duration"
+  "decompress" {>= "0.9.0" & < "1.0.0"}
+  "checkseum"
+  "metrics" {>= "0.2.0"}
+  "metrics-lwt" {>= "0.2.0"}
+  "metrics-influx" {>= "0.2.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depexts: [
+  ["libnl-3-dev" "libnl-route-3-dev"] {os-family = "debian"}
+  ["libnl3" "libnl3-devel"] {os-family = "centos"}
+]
```

Comment:
Would it be possible to move this into a separate `conf-libnl` package?

Reply:
I will do that.

libnl3 is divided into a handful of libraries. On debian they are packaged separately while on centos they seem to all live in libnl3-devel. Would you advice creating conf-libnl3, conf-libnl-route3, etc. or just one conf-libnl3 with all of them?

Reply:
thanks @reynir -- I suggest a single `conf-libnl3` which includes the needed packages. when some other package needs a subset, we can always split conf-libnl3 into multiple packages.